client/templates/app/signup/signup.html

```diff
@@ -1,7 +1,8 @@
-<template name="publicSignup">  
+<template name="publicSignup">
   <div class="row">
+  {{> atForm}}
+    <!--
     <form class="col s6 offset-s3 signup-form">
```

Comment:
Right, all that's really needed is {{> atForm}}.  I've removed it and recommitted.

I only left so much stuff in (commented out) because I wasn't 100% confident everybody was going to be on-board with useraccounts packages.